modules/drivers/mongo/src/metabase/driver/mongo/execute.clj

```diff
@@ -168,24 +168,12 @@
     (finally
       (.close cursor))))
 
-(defn- parse-query-string
```

Comment:
Moved to the query processor since it has to parse nested native queries in text form.